manifests/plugin/apache.pp

```diff
@@ -1,17 +1,23 @@
 # https://collectd.org/wiki/index.php/Plugin:Apache
 class collectd::plugin::apache (
-  $ensure         = present,
-  $manage_package = $true,
-  $instances      = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' } },
-  $interval       = undef,
+  $ensure                  = present,
+  $manage_package          = $true,
+  $instances               = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' } },
+  $interval                = undef,
+  $package_install_options = $collectd::params::package_install_options,
```

Comment:
You are correct, we should use `$collectd::package_install_options`. If you need to specify `install_options` it is very likely you would need it for everything, so defaulting to that is a good idea.